nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-client-dto/src/main/java/org/apache/nifi/web/api/dto/PortDTO.java

```diff
@@ -171,4 +172,18 @@ public void setValidationErrors(Collection<String> validationErrors) {
         this.validationErrors = validationErrors;
     }
 
+
+    /**
+     * @return whether this port can be accessed remotely via Site-to-Site protocol.
+     */
+    @ApiModelProperty(
+            value = "Whether this port can be accessed remotely via Site-to-Site protocol."
+    )
+    public Boolean isAllowRemoteAccess() {
```

Comment:
is this intentionally a non primitive type? if so the method should read as getAllowRemoteAccess.